components/FrontPageCard/FrontPageCard.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Block,
+  ColorDesignTokens,
+  Heading,
+  Link,
+  suomifiDesignTokens,
+} from 'suomifi-ui-components';
+import styles from './FrontPageCard.module.scss';
+
+interface FrontPageCardProps {
+  imgSrc: string;
+  bgColor: keyof ColorDesignTokens;
+  heading: string;
+  linkText: string;
+  linkHref: string;
+}
+
+const FrontPageCard: React.FunctionComponent<FrontPageCardProps> = ({
+  imgSrc,
+  bgColor,
+  heading,
+  linkText,
+  linkHref,
+}) => {
+  return (
+    <Block variant="div" className={styles.frontPageCard}>
+      <Block
+        className={styles.upper}
+        style={{ backgroundColor: suomifiDesignTokens.colors[bgColor] }}
+      >
+        <img src={imgSrc} alt="" />
```

Comment:
next.js seemed to prefer its own Image component instead of <img>. Does it make a difference?

Reply:
Next.js <Image> behaves weirdly since it requires an explicit width and height if using src as a string (like we need to do in this prop-based scenario)

It was easier to use the regular <img> in this case